upf/upf_pfcp_server.h

```diff
@@ -209,11 +212,14 @@ pfcp_msg_pool_add (pfcp_server_main_t * psm, pfcp_msg_t * m)
   msg = pfcp_msg_pool_get (psm);
   clib_memcpy_fast (msg, m, sizeof (*m));
   msg->is_valid_pool_item = 1;
+  msg->loc[0] = 0;
   return msg;
 }
 
+#define pfcp_msg_pool_put(psm, m) do { snprintf((m)->loc, 128, "%s:%d %s", __FILE__, __LINE__, __FUNCTION__);  _pfcp_msg_pool_put(psm, m); } while (0)
```

Comment:
shouldn't that be restricted to debug builds only?

Reply:
That was not meant to be committed at all, thanks for spotting it! On the second thought, maybe indeed we should keep that for debug builds. Bugs such as one that PR fixes are usually rare and not always easy to reproduce, and if another problem like this one is hit, we better see how exactly it happened on the first occurrence.

Will fix, thanks